executor/index_lookup_join_test.go

```diff
@@ -47,49 +47,15 @@ func (s *testSuite1) TestIndexJoinUnionScan(c *C) {
 	tk.MustExec("insert into t1 values(2,2)")
 	tk.MustExec("insert into t2 values(2,2,2), (3,3,3)")
 	// TableScan below UnionScan
-	tk.MustQuery("explain select /*+ TIDB_INLJ(t1, t2)*/ * from t1 join t2 on t1.a = t2.id").Check(testkit.Rows(
```

Comment:
The purpose of this test is to verify that correct results are returned by the query, so we can just remove these explain output since they are pretty fragile.